storage/s3-sdk/README.md

```diff
@@ -32,11 +32,13 @@ Install [Maven](http://maven.apache.org/).
 
 ## Test Sample
 
+1. Provide a service account which can be used to generate HMAC Key for the scope of the tests
+   
+   * GOOGLE_APPLICATION_CREDENTIALS=[PATH_TO_SERVICE_ACCOUNT_JSON]
+
 1. Set the following environment variable with the default project for Interoperable Storage Access Keys.
 
    * GOOGLE_CLOUD_PROJECT_S3_SDK=[GOOGLE_PROJECT_ID]
```

Comment:
Does this still need to be a separate project? For python, I switched to using the project used by the HMAC samples. (On the other hand I think these are in a separate repo for Java...)

Reply:
This variable appears to actually only be used as the name of the bucket for `storage.s3sdk.ListGcsObjectsTest`. Happy to rename this if @GoogleCloudPlatform/java-samples-reviewers are okay with the rename.

Reply:
Yea - that looks horrible, shouldn't it just use the standard project?

Reply:
Before this change, the project was using a specific project and bucket which has some preconfigured HMAC credentials so it was branching off this way.

Since this change updatings things to use an HMAC key per test class, this is less necessary. I've updated the test to use a fallback of the projectId from the service account used to create the hmac key, and I've renamed this environment variable so that the name is descriptive of what it's actually used for.